toolchain/src/integration-test/resources/queries/firstRuleViolated.sparql

```diff
@@ -0,0 +1,16 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

```

Comment:
Just a general question: Should we include all prefix declarations even though not all of them are actually used?

Reply:
They are not required. I just copy-pasted the code from the `SPARQL-queries` directory and did a few modifications. The example queries have them so that users do not have to look them up (I guess). This argument does not apply to tests, so I will remove it.

Reply:
They are removed now.

Reply:
:+1: